rubygem-azure-core/rubygem-azure-core.spec

```diff
@@ -0,0 +1,87 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Generated from azure-core-0.1.1.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name azure-core
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.1
+Release: 1%{?dist}
+Summary: Core library to be consumed by Ruby SDK gems
+Group: Development/Languages
+License: ASL 2.0
+URL: http://github.com/Azure/azure-ruby-asm-core
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
```

Comment:
It was testing the gem but not picking up errors from `tito release`, Jenkins job now fixed!